plugins/javadoc/src/main/kotlin/org/jetbrains/dokka/javadoc/pages/utils.kt

```diff
@@ -1,15 +1,39 @@
 package org.jetbrains.dokka.javadoc.pages
 
 import org.jetbrains.dokka.model.*
+import org.jetbrains.dokka.utilities.htmlEscape
 
-internal fun JavadocFunctionNode.getAnchor(): String =
-    "$name(${parameters.joinToString(",") {
-        when (val bound = if (it.typeBound is Nullable) it.typeBound.inner else it.typeBound) {
-            is TypeConstructor -> listOf(bound.dri.packageName, bound.dri.classNames).joinToString(".")
-            is TypeParameter -> bound.name
-            is PrimitiveJavaType -> bound.name
-            is UnresolvedBound -> bound.name
-            is JavaObject -> "Object"
-            else -> bound.toString()
-        }
-    }})"
\ No newline at end of file
+/**
+ * Returns an unencoded, unescaped function anchor.
+ *
+ * Should be URL encoded / HTML escaped at call site,
```

Comment:
But you do not encode it in the PR.

Reply:
Yes, what's the problem? This method does no encoding/escaping, so the result should be encoded/escaped in the places where it is called if there's need for it ("depending on usage" from the line below)

Reply:
Sorry, this comment should be on the next line. You don't encode/escape a returned value at call site,
e.g. https://github.com/Kotlin/dokka/blob/53a57b0e4dd6abcb7b6d68be0c5983050906e0d8/plugins/javadoc/src/main/kotlin/org/jetbrains/dokka/javadoc/renderer/JavadocContentToTemplateMapTranslator.kt#L125  and https://github.com/Kotlin/dokka/blob/53a57b0e4dd6abcb7b6d68be0c5983050906e0d8/plugins/javadoc/src/main/kotlin/org/jetbrains/dokka/javadoc/renderer/JavadocContentToTemplateMapTranslator.kt#L170
Have a look at templates before changing.

Reply:
Yeah, for sure, I don't. We discussed why it's not as trivial in the [conversation above](https://github.com/Kotlin/dokka/pull/2813#discussion_r1072293998). 

TLDR: this PR is not about encoding/escaping, it's about fixing leaking details due to `.toString()`. The KDoc for the function has little to do with where/how it's actually used, I documented the behaviour of the function.

Reply:
So can you open an issue for this?

Reply:
#2854